### ISSUE TITLE
ListLoadBalancerRuleInstances output does not match

### ISSUE THREAD

**Issue by ngtuna** [xanzy/go-cloudstack#54]
Hi,

I tried to call api `listLoadBalancerRuleInstances` on CS 4.8. The output was more detailed like this:
```
{  
   "listloadbalancerruleinstancesresponse":{  
      "count":1,
      "lbrulevmidip":[  
         {  
            "loadbalancerruleinstance":{  
               "id":"9f0191b0-f976-4bec-8c89-6d341d1123b6",
               "name":"coretest",
               "displayname":"coretest",
               "account":"admin",
               "userid":"8cb7dbf6-0ef7-11e6-aafe-0800271998f3",
               "username":"admin",
               "domainid":"f76d7c22-0ef6-11e6-aafe-0800271998f3",
               ...
```
However, the response object in `go-cloudstack` is too simple as below and thus doesn't fit the real output:
```
type ListLoadBalancerRuleInstancesResponse struct {
	Count                     int                         `json:"count"`
	LoadBalancerRuleInstances []*LoadBalancerRuleInstance `json:"loadbalancerruleinstance"`
}

type LoadBalancerRuleInstance struct {
	Lbvmipaddresses          []string `json:"lbvmipaddresses,omitempty"`
	Loadbalancerruleinstance string   `json:"loadbalancerruleinstance,omitempty"`
}

```
Please help to update the `ListLoadBalancerRuleInstancesResponse ` object.

Thanks,

**Comment by svanharmelen** [comment]
@ngtuna thanks for the report!

At first sight the response given by `go-cloudstack` seems correct as per the API [docs](http://cloudstack.apache.org/docs/api/apidocs-4.8/root_admin/listLoadBalancerRuleInstances.html) and the `listApis` output:

```json
{
  "description": "List all virtual machine instances that are assigned to a load balancer rule.",
  "isasync": false,
  "name": "listLoadBalancerRuleInstances",
  "params": [
    {
      "description": "the ID of the load balancer rule",
      "length": 255,
      "name": "id",
      "related": "listIpForwardingRules,listPortForwardingRules,updatePortForwardingRule,createPortForwardingRule",
      "required": true,
      "type": "uuid"
    },
    {
      "description": "true if load balancer rule VM IP information to be included; default is false",
      "length": 255,
      "name": "lbvmips",
      "required": false,
      "type": "boolean"
    },
    {
      "description": "true if listing all virtual machines currently applied to the load balancer rule; default is true",
      "length": 255,
      "name": "applied",
      "required": false,
      "type": "boolean"
    },
    {
      "description": "",
      "length": 255,
      "name": "page",
      "required": false,
      "type": "integer"
    },
    {
      "description": "",
      "length": 255,
      "name": "pagesize",
      "required": false,
      "type": "integer"
    },
    {
      "description": "List by keyword",
      "length": 255,
      "name": "keyword",
      "required": false,
      "type": "string"
    }
  ],
  "related": "listLoadBalancerRuleInstances",
  "response": [
    {
      "description": "the user vm set for lb rule",
      "name": "loadbalancerruleinstance",
      "type": "uservmresponse"
    },
    {
      "description": "IP addresses of the vm set of lb rule",
      "name": "lbvmipaddresses",
      "type": "list"
    }
  ]
}
```

Will need to look a little deeper in order to see if/how we can fix this. The `cloudstack` package is fully generated using the `listApis` output, so we would probably need to add some custom code for this specific endpoint...

**Comment by svanharmelen** [comment]
Fixed by PR #55

**Comment by ngtuna** [comment]
Awesome. Thanks @svanharmelen
